db/migrate/20200512170845_add_user_to_curated_cluster.rb

```diff
@@ -1,5 +1,6 @@
 class AddUserToCuratedCluster < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
```

Comment:
have you investigated why rails asked you to put this line in there?

Reply:
Still looking into this. The message came up when trying to commit without line 2 being included. I'm reading through this article that goes over why the line would want to be included:

https://redpanthers.co/disabling-transaction-block-during-migration/